app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/accounts/homepage/HomePageController.scala

```diff
@@ -0,0 +1,91 @@
+import uk.gov.hmrc.cgtpropertydisposalsfrontend.views
+import uk.gov.hmrc.play.bootstrap.controller.FrontendController
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class HomePageController @Inject()(
+                                val authenticatedAction: AuthenticatedAction,
+                                val sessionDataAction: SessionDataAction,
+                                errorHandler: ErrorHandler,
+                                sessionStore: SessionStore,
+                                cc: MessagesControllerComponents,
+                                manageYourDetailsPage: views.html.account.manage_your_details,
+                                homePage: views.html.account.home,
+                                privateBetaHomePage: views.html.account.home_private_beta,
+                                detailUpdatedPage: views.html.account.details_updated,
+                                signedOutPage: views.html.account.signed_out
+                              )(implicit viewConfig: ViewConfig, ec: ExecutionContext)
+  extends FrontendController(cc)
+    with WithAuthAndSessionDataAction
+    with SessionUpdates
+    with Logging {
+
+  // homepage for after private beta: includes functionality to do with returns
+  def homepage(): Action[AnyContent] = authenticatedActionWithSessionData.async {
+    implicit request: RequestWithSessionData[AnyContent] =>
+      withSubscribedUser(request) { (_, subscribed) =>
+        Future.successful(Ok(homePage(subscribed.subscribedDetails)))
+      }
+  }
+
+  // homepage for private beta: does not include any functionality to do with returns. This will
+  // eventually be removed when we move out of private beta
+  def privateBetaHomepage(): Action[AnyContent] = authenticatedActionWithSessionData.async {
+    implicit request: RequestWithSessionData[AnyContent] =>
+      withSubscribedUser(request) { (_, subscribed) =>
+        Future.successful(Ok(privateBetaHomePage(subscribed.subscribedDetails)))
+      }
+  }
+
+  def startNewReturn(): Action[AnyContent] = authenticatedActionWithSessionData.async { implicit request =>
+    withSubscribedUser(request) { (_, _) =>
+    request.userType match  {
+      case Some(UserType.Individual) =>
+        Future.successful(Redirect(controllers.returns.triage.routes.CanTheyUseOurServiceController.whoIsIndividualRepresenting()))
+
+      case other =>
+        logger.warn(s"Start a new return for user type: $other is not supported")
+        errorHandler.errorResult()
+    }
+    }
+  }
+
+  private def withSubscribedUser(request: RequestWithSessionData[_])(
+    f: (SessionData, Subscribed) => Future[Result]
+  ): Future[Result] =
+    request.sessionData.flatMap(s => s.journeyStatus.map(s -> _)) match {
+      case Some((s: SessionData, r: Subscribed)) =>
+        f(s, r)
+      case _ =>
+        Future.successful(SeeOther(uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.routes.StartController.start().url))
+    }
```

Comment:
import the controller package to make the line shorter - same elsewhere in this file

Reply:
done